src/miner.cpp

```diff
@@ -829,15 +841,13 @@ namespace pos {
     }
 
     template <typename F>
-    bool Staker::withSearchInterval(F&& f) {
-        const int64_t nTime = GetAdjustedTime(); // TODO: SS GetAdjustedTime() + period minting block
+    void Staker::withSearchInterval(F&& f) {
+        // Mine up to max future minus 10 second buffer
+        nFutureTime = GetAdjustedTime() + (MAX_FUTURE_BLOCK_TIME_DAKOTACRESCENT - 10);
```

Comment:
```suggestion
        nFutureTime = GetAdjustedTime() + (MAX_FUTURE_BLOCK_TIME_DAKOTACRESCENT - 5);
```

Reply:
Let's do 5 secs. I think it should be safe enough while not giving anyone any significant extra edge.